src/components/header.js

```diff
@@ -2,41 +2,46 @@ import { Link } from "gatsby"
 import PropTypes from "prop-types"
 import React from "react"
 
+import Container from 'react-bootstrap/Container';
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+
+import "./header.scss"
+import logo from "../images/oo-logo-combined.svg"
+
+
 const Header = ({ siteTitle }) => (
-  <header
-    style={{
-      background: `rebeccapurple`,
-      marginBottom: `1.45rem`,
-    }}
-  >
-    <div
-      style={{
-        margin: `0 auto`,
-        maxWidth: 960,
-        padding: `1.45rem 1.0875rem`,
-      }}
-    >
-      <h1 style={{ margin: 0 }}>
-        <Link
-          to="/"
-          style={{
-            color: `white`,
-            textDecoration: `none`,
-          }}
-        >
-          {siteTitle}
-        </Link>
-      </h1>
-    </div>
-  </header>
+    <header>
+        <Navbar bg="white" variant="light" fixed="top" expand="md"  className="d-flex flex-column">
+                <div id="logo-row" className="d-flex flex-row align-items-center justify-content-between">
+                    <div id="logo" className="mx-sm-auto ml-lg-4">
+                        <Link to="/"><img src={logo} alt="Logo" /></Link>
+                    </div>
+                    <div id="tagline" className="d-none d-lg-block">
+                        Volunteers providing transparent access to Oakland Unified School District budget & data
+                    </div>
+                    <Navbar.Toggle aria-controls="basic-navbar-nav" />
+                </div>
+                <div id="menu-row">
+                    <Navbar.Collapse id="basic-navbar-nav" className="flex-grow-1 text-right">
+                        <Nav className="menu mx-auto flex-nowrap">
+                            <Nav.Item>Home</Nav.Item>
+                            <Nav.Item>Departments</Nav.Item>
+                            <Nav.Item><Link to="/page-2/">Contact</Link></Nav.Item>
+                        </Nav>
+                    </Navbar.Collapse>
+                </div>
+        </Navbar>
+    </header>
 )
 
 Header.propTypes = {
-  siteTitle: PropTypes.string,
+    siteTitle: PropTypes.string,
```

Comment:
It looks to me like `siteTitle` is not actually used anywhere, so you should be able to remove it.